app/javascript/utils/distributions_and_transfers.js

```diff
@@ -106,3 +106,58 @@ $(function() {
     request_storage_location_and_populate_item(default_item);
   });
 });
+
+$(document).ready(function (e) {
+  $(document).on('change', '.dist_delivery_method', hide_show_shipping_cost);
+  show_shipping_cost();
+});
+
+function show_shipping_cost()
+{
+  var delivery_method_radio = $(".dist_delivery_method");
+  var shipping_cost_div = $(".distribution_shipping_cost");
+  var shipped = false;
+
+  if ( delivery_method_radio.length>0 && shipping_cost_div.length>0 ) {
+    $.each(delivery_method_radio, function(index, delivery_method) {
+       if(delivery_method.checked == true && delivery_method.value == "shipped")
+       {
+         shipped=true;
+       }
+    });
```

Comment:
If you're already using jQuery you can get the value much more easily:

```
$('input[name="delivery_method"]:checked').val()
```

Reply:
done